packages/babel-make-styles/__fixtures__/object-mixins/mixins.ts

```diff
@@ -9,3 +9,8 @@ export const gridStyles = (gridGap: string): MakeStyles => ({
   display: 'grid',
   gridGap,
 });
+
+export const typography: Record<'text' | 'header', MakeStyles> = {
```

Comment:
should we also add an example with namespace export ?

Reply:
Not really, as we are testing `makeStyles()` call itself. Even this was throwing before:

```tsx
const typography = {
  header: { color: 'red' }
}

export const useStyles = makeStyles({
  root: flexStyles,
  header: typography.header,
})
```